unlock_indicator.c

```diff
@@ -6,26 +6,40 @@
  * See LICENSE for licensing information
  *
  */
+#include <cairo.h>
+#include <cairo/cairo-xcb.h>
+#include <ev.h>
+#include <math.h>
 #include <stdbool.h>
-#include <stdlib.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
-#include <math.h>
 #include <xcb/xcb.h>
-#include <ev.h>
-#include <cairo.h>
-#include <cairo/cairo-xcb.h>
 
+#include "dpi.h"
 #include "i3lock.h"
-#include "xcb.h"
-#include "unlock_indicator.h"
 #include "randr.h"
-#include "dpi.h"
+#include "unlock_indicator.h"
+#include "xcb.h"
 
 #define BUTTON_RADIUS 90
 #define BUTTON_SPACE (BUTTON_RADIUS + 5)
 #define BUTTON_CENTER (BUTTON_RADIUS + 5)
 #define BUTTON_DIAMETER (2 * BUTTON_SPACE)
+/* Color constants and stuff */
```

Comment:
Let's remove this comment and maybe instead ive the constants more descriptive names as to what they represent.

Reply:
Will do